docs/Use_cases/Air_quality_monitoring_use_case.md

```diff
@@ -0,0 +1,229 @@
+- An active Microsoft Power BI account.
+- An OpenWeatherMap account with a free API token.
+
+## Creating and configuring an Azure IoT hub and storage account
+
+First you need to add a new IoT hub and a storage account in Azure. Here's how to do it:
+
+### Creating an IoT hub
+
+1. In your Azure portal home view, go to **IoT Hub** and select **Add**.
+![Adding an IoT hub](images/hub_add.png "Add iot hub")
+    - In the **Basics** tab:
+        - select your subscription and resource group,
+        - pick your region,
+        - provide a name for your IoT hub.
+![IoT hub basics tab](images/hub_basics.png "Iot hub basics")
+    - In the **Management** tab:
+        - in **Pricing and scale tier** select,
+        - optionally, turn off **Defender for IoT**.
+    - In the **Review + create** tab, click **Create**.
+![IoT hub management tab](images/hub_create.png "Iot hub management")
+
+### Creating a storage account
+
+While your new IoT hub is deploying, you can add a new storage account:
+
+1. In the Azure portal, go to **Storage accounts** and select **Add**.
+
+   - In the **Basics** tab:
+       - select your subscription and resource group,
+       - provide a name for your storage account,
+       - pick your location.
+   - In the **Review + create** tab, click **Create**.
+
+## Configuring the Azure IoT Hub integration extension
+
+Once the deployments are complete, go to Coiote DM to set up the Azure IoT Hub extension.
+If you haven't done this yet, please follow the [instruction for the Azure IoT Hub integration configuration](../Configuring_Azure_IoT_Hub_integration_extension).
+
+## Adding and connecting LwM2M air quality meter simulators to Coiote DM and Azure IoT Hub
+1. Go to your Azure IoT Hub and add new devices:
+    - Under **Explorers**, select **IoT Devices** and click **+ New**.
+    - Provide the name for your first device: ``air-quality-meter-example-0``.
+    - Click **Save**.
+    - Repeat the procedure for the other 5 devices (increase the number included in the device name).
+2. Go to Coiote DM and sync the previously added devices:
+    - In **Device inventory**, select **Sync with IoT platform -> Azure IoT Hub**.
+    - In the pop-up, click **Sync devices**.
+    - Devices should then be visible in **Device inventory**   
+3. Go to your command line and register the device simulators:
+    - Paste and run the following command to create a container group:
+        ```
+         az container create -g coiote-dm-experiments --name air-quality-meter-example-0 --image avsystemcom/air-quality-meter-example --environment-variables DEVICEID-air-quality-meter-example-0 SERVER_ADDRESS=eu.iot.avsystem.cloud OPEN_WEATHER_API_TOKEN=exampletoken
+
+        ```
+
+        !!! note
+            Remember to change the command parameters accordingly so that they are in line with your naming and credentials.
+
+     - once the command is executed, you should see a JSON payload that describes the content of the container instance.
```

Comment:
Show the example result - or screen. The user needs a way to check if their result is valid.

Reply:
ok

Reply:
Blurred sha256 doesn't make any sense here. It is not a secret, this is just a hash.